aries-backchannels/acapy/acapy_backchannel.py

```diff
@@ -218,7 +218,7 @@ def get_agent_args(self):
             ("--label", self.label),
             # "--auto-ping-connection",
             # "--auto-accept-invites",
-            # "--auto-accept-requests",
+            "--auto-accept-requests",
```

Comment:
@TimoGlastra Did you mean to change this to have --auto-accept-requests on? It seems to be causing at least one failure. https://allure.vonx.io/api/allure-docker-service/projects/afgo-b-acapy/reports/latest/index.html?redirect=false#behaviors/b42ec1e2628bcda077660120786966a4/865c2ca749632e9b/

Reply:
This isn't documented really great, but there is mention here of a way to pass a configuration file to the acapy start command in the backchannel using `AGENT_CONFIG_FILE`. https://github.com/hyperledger/aries-agent-test-harness/blob/main/README.md#using-aath-agents-as-services

So using a mediation config file, we could either have a new runset with Mediation, or better yet, just add another run section in the workflow file before uploading the results to allure. Not sure if there would have to be changes to the action or not to support this. Maybe another action that just runs tests instead of building everything all over again. 

The base tests require all auto options off on aca-py.

Reply:
I think we could solve this using the `/agent/command/agent/start` endpoint which would be a lot simpler than adding more runsets, as we can just do it dynamically during runtime. I think we should aim for as much dynamic configuration as it's becoming more and more complex to run the tests with all agents having a very specific support matrix for features.

Reply:
We still have the failing test mentioned in issue #456, we should make a decision on this and implement. I still lean on using the same workflow with a following run test step. This then is not another run set, but just another execution of more tests in the same runset. 

Another way we can approach this is to actually have another agent run like an Auto_Acme, and if that agent is needed the the test scenarios for mediation can begin something like,
```gherkin
    Given we have "2" agents
      | name | role      |
      | Auto_Acme | responder |
      | Bob  | requester |
```
We can come up with a better name than auto_acme, but you get the point. This way Acme stays the same and this new agent comes into play when needed. Thoughts?

Reply:
I took a stab at this but got distracted by other tasks, will try to finish. The approach I've taken is to restart the agent using different parameters.

Re your comment:

> There is also the argument that once we start up an agent with different parameters, the test environment has changed, and would warrant another runset to differentiate. Combining a newly instantiated agent into an already running suite, may cause issues with test order. You'd have to be sure these tests are executed last or resets the agent into it's previous state before continuing.

This is already being used for other tests and whenever the `@UsesCustomParameters` tag is available the agents will be restarted and reset to their initial startup configuration. This allows to start an agent with specified configuration before a test and restore it afterwards.

I like this approach because it's a bit more flexible IMO than adding e.g. the Auto_Acme agent. What if we need just a slightly different configuration next week? With dynamic configuration this would be as simple as modifying the agent startup configuration, without adding another agent to the test matrix.